### PR TITLE
Add Thai localization

### PR DESCRIPTION
**Pull request by naijab**
🇹🇭

I love this app so much. :D

### CODE REVIEW

File: Maccy/Preferences/th.lproj/AppearancePreferenceViewController.strings [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+"0h4-CZ-fwT.title" = "เมนูไอคอน";
+
+/* Class = "NSTextFieldCell"; title = "Press ⌘, to open preferences when footer is hidden."; ObjectID = "0tN-4z-p8T"; */
+"0tN-4z-p8T.title" = "กด ⌘, เพื่อเปิดการตั้งค่าเมื่อส่วนท้ายซ่อน";
+
+/* Class = "NSButtonCell"; title = "Show recent copy next to menu icon"; ObjectID = "2vE-8q-6tq"; */
+"2vE-8q-6tq.title" = "แสดงการคัดลอกล่าสุดไปยังเมนูไอคอน";
+
+/* Class = "NSMenuItem"; title = "Cursor"; ObjectID = "Ccp-5J-8nw"; */
+"Ccp-5J-8nw.title" = "Cursor";
```
Comment (p0deje) [inline Maccy/Preferences/th.lproj/AppearancePreferenceViewController.strings:12]: Is this okay?

File: Maccy/Preferences/th.lproj/PinsPreferenceViewController.strings [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+/* Class = "NSTextFieldCell"; title = "Text Cell"; ObjectID = "bdM-5c-VDK"; */
+"bdM-5c-VDK.title" = "Cell ข้อความ";
+
+/* Class = "NSTableColumn"; headerCell.title = "Title"; ObjectID = "bsY-ZI-mU2"; */
+"bsY-ZI-mU2.headerCell.title" = "หัวข้อ";
+
+/* Class = "NSTextFieldCell"; title = "You can customize the title and hotkey of every pinned item.\nTo edit, double-click it and enter new value.\nFor key, you can only use one of the following values: %@."; ObjectID = "jBl-gB-eEV"; */
+"jBl-gB-eEV.title" = "คุณสามารถตั้งค่าหัวข้อ และคีย์ลัดสำหรับรายการปักหมุดได้.\nเพื่อแก้ไข, double-click รายการนั้นแล้ว แก้ไขค่าใหม่.\nสำหรับคีย์, คุณสามารถใช้ใช้ได้ตามนี้: %@.";
```
Comment (p0deje) [inline Maccy/Preferences/th.lproj/PinsPreferenceViewController.strings:9]: Is "double-click" okay?

Comment (p0deje) [inline Maccy/Preferences/th.lproj/PinsPreferenceViewController.strings:9]: Is "double-click" okay?

File: Maccy/Preferences/th.lproj/AdvancedPreferenceViewController.strings [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+/* Class = "NSTextFieldCell"; title = "Automatically remove all unpinned items before quitting the application."; ObjectID = "LW1-8W-pFI"; */
+"LW1-8W-pFI.title" = "ล้างรายการที่ไม่ปักหมุดก่อนออกแอปพลิเคชัน";
+
+/* Class = "NSButtonCell"; title = "Turn off"; ObjectID = "RA0-eh-Udt"; */
+"RA0-eh-Udt.title" = "ปิด";
+
+/* Class = "NSTextFieldCell"; title = "Maccy takes an active application focus when the popup is shown and gives focus back when it's closed. The focus is needed for advanced macOS input handling such as Chinese helper or accent menu. Enabling this option would prevent Maccy from taking focus. There are certain situations when this can be useful: for example, pasting to a text editor that auto-formats code when it loses focus."; ObjectID = "Y8y-xQ-lFm"; */
+"Y8y-xQ-lFm.title" = "Maccy จะทำการกับแอป Active อยู่. จำเป็นต้องมีตั้งค่าสำหรับการจัดการ input ของ macOS ในระดับสูง เช่น ตัวช่วยการพิมพ์ภาษาจีนหรือเมนูอื่นๆ. เปิดตัวเลือกนี้ป้องกันไม่ให้ Maccy หลุดการทำงาน. มีบางสถาการณ์ที่จะเป็นประโยชน์ เช่น การวางลงแอปแก้ไขข้อความ เพื่อป้องกันการหลุดการทำงาน";
```
Comment (p0deje) [inline Maccy/Preferences/th.lproj/AdvancedPreferenceViewController.strings:18]: Is "Active" and "input" okay?

Comment (p0deje) [inline Maccy/Preferences/th.lproj/AdvancedPreferenceViewController.strings:18]: Is "Active" and "input" okay?

**Comment by p0deje** [comment]
Hi, thank you for the contribution, I've left few comments and I'll merge as soon as you reply.

File: Maccy/Preferences/th.lproj/AppearancePreferenceViewController.strings (hunk 1 above)
Comment (naijab) [reply Maccy/Preferences/th.lproj/AppearancePreferenceViewController.strings:12]: for Thai lang some word is difficult to use we use same sound of original

File: Maccy/Preferences/th.lproj/PinsPreferenceViewController.strings (hunk 2 above)
Comment (naijab) [reply Maccy/Preferences/th.lproj/PinsPreferenceViewController.strings:9]: double click is easy understand for Thai. use this for normal

File: Maccy/Preferences/th.lproj/AdvancedPreferenceViewController.strings (hunk 3 above)
Comment (naijab) [reply Maccy/Preferences/th.lproj/AdvancedPreferenceViewController.strings:18]: yes

Comment (naijab) [reply Maccy/Preferences/th.lproj/AdvancedPreferenceViewController.strings:18]: yep

**Pull request merged**